generator/testdata/rust/gclient/golden/iam/v1/src/client.rs

```diff
@@ -97,15 +97,33 @@ impl IAMPolicy {
     /// existing policy.
     ///
     /// Can return `NOT_FOUND`, `INVALID_ARGUMENT`, and `PERMISSION_DENIED` errors.
-    pub fn set_iam_policy(&self) -> crate::builders::SetIamPolicy {
+    pub fn set_iam_policy<
+        IntoResource,
+    >(
+        &self,
+        resource: IntoResource,
```

Comment:
This signature is a little hard to read imo with the indirection of the where clause. How would you fell about:

```
pub fn set_iam_policy(&self, resource: impl Into<String>) -> crate::builders::SetIamPolicy { ... }
```

Reply:
Done.